serverless/event_sources/serverless-containersource.adoc

```diff
@@ -0,0 +1,11 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-containersource"]
+= Using container sources
+include::modules/common-attributes.adoc[]
+:context: serverless-containersource
+
+toc::[]
+
+Container sources create a container image that generates events and sends events to a sink. You can use a container source to create a custom event source.
```

Comment:
This is likely my ignorance. You state that you _can_ use a container source to create a custom event source. That makes it sound optional, or requires an additional step. Is that the case. If not, should this state:  _The container source creates a custom event source._ (or something similar)